README/image.html

```diff
@@ -0,0 +1,19 @@
+<!-- Use this HTML to generate the output sample image -->
```

Comment:
I'm not familiar with gradle / the build process - will these files be included as part of the files that are used to run the command line tool? Should they be ignored?

Reply:
This file is a helper file to generate the screenshot of the output. It's not used for anything else. It's a manual process:
1 - Get the screenshot
2 - Put in the directory
3 - Change the HTML to point to your new screenshot
4 - Open the HTML in the browser
5 - Take a screenshot of the screenshot with the shadow and all the jazz
6 - Put the new screenshot in this directory to overwrite `output_sample.png`
7 - Delete all other files in this directory but the HTML and the output sample image

It's not a complex process, it's just complex to describe. :-)

I'll add this as a comment to the HTML.

To answer you question, the HTML cannot be ignored.